eessi_bot_event_handler.py

```diff
@@ -52,6 +52,7 @@
         config.BUILDENV_SETTING_BUILD_JOB_SCRIPT,                  # required
         config.BUILDENV_SETTING_BUILD_LOGS_DIR,                    # optional+recommended
         config.BUILDENV_SETTING_BUILD_PERMISSION,                  # optional+recommended
+        config.BUILDENV_SETTING_CLONE_GIT_REPO_VIA,                # optional
```

Comment:
If not commented, the setting is 'required'.

```suggestion
        #config.BUILDENV_SETTING_CLONE_GIT_REPO_VIA,                # optional
```